src/Compilers/VisualBasic/Portable/LanguageVersion.vb

```diff
@@ -89,7 +89,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Case LanguageVersion.Latest
                     Return LanguageVersion.VisualBasic16_9
                 Case LanguageVersion.Default
```

Comment:
```suggestion
                Case LanguageVersion.Latest,
                     LanguageVersion.Default
```

Worth grouping both cases together so that future updates don't miss one of them?

Reply:
I'd prefer not. If we then need to update `Latest` independently again, it's a bigger change.

Reply:
@333fred I thought the goal of the change is to keep "default" to **always** be "latest". If that's not the case, I'm interested to understand more on why the change is needed. Specially that the [current documentation](https://docs.microsoft.com/en-us/dotnet/visual-basic/language-reference/configure-language-version) mentions:

> The special strings default and latest resolve to the latest major and minor language versions installed on the build machine, respectively.

The latest major is `16.0`, and the latest minor is `16.9`. I was actually going to open a doc PR after this is merged to update this sentence to:

> The special strings default and latest resolve to the latest language version installed on the build machine.